install_files/ansible-base/securedrop-configure.yml

```diff
@@ -60,6 +60,7 @@
 
     - name: securedrop_app_gpg_fingerprint
       prompt: Full fingerprint for the SecureDrop Application GPG Key
+      default: ' '
```

Comment:
Shouldn't this be `''`, not `' '`? I suppose this would still fail validation, but using an empty string (i.e. `''`) seems much more intuitive to me.